ci-operator/step-registry/openstack/install/multiplenetworks/openstack-install-multiplenetworks-workflow.yaml

```diff
@@ -0,0 +1,35 @@
+workflow:
+  as: openstack-install-multiplenetworks
+  steps:
+    pre:
+    # from chain: ipi-conf-openstack
+    - ref: ipi-conf-openstack
+    - ref: openstack-conf-createfips
+    - ref: openstack-conf-createadditionalnetworks
+    - ref: openstack-conf-generateconfig
+    - ref: openstack-conf-creatednsrecords
+    - ref: ipi-conf-etcd-on-ramfs
+    - ref: ipi-conf-openstack-precheck
+    - ref: ipi-install-monitoringpvc
+    # From chain: ipi-install
+    - ref: ipi-install-rbac
+    - ref: ipi-install-install
+
+    test:
+#      - ref: openshift-e2e-test
+    post:
+    - chain: ipi-deprovision
+    # from - chain: ipi-deprovision-openstack
+    - ref: openstack-deprovision-deletednsrecords
+    - ref: openstack-deprovision-deletefips
+    - ref: openstack-deprovision-deleteadditionalnetworks
+    env:
+      OPENSTACK_EXTERNAL_NETWORK: "external"
+      OS_CLOUD: "openstack-cloud"
+      OPENSTACK_COMPUTE_FLAVOR: "m1.s2.xlarge"
```

Comment:
Remove OS_CLOUD, OPENSTACK_EXTERNAL_NETWORK and OPENSTACK_COMPUTE_FLAVOR unless you want to override the default values for the cloud defined in openstack-conf-resourcenames